client/app/dedicated/server/dedicated-server.service.js

```diff
@@ -2152,4 +2154,20 @@ angular
     this.isAutoRenewable = function isAutoRenewable(productId) {
       return this.getSelected(productId).then(server => moment(server.expiration).diff(moment().date(), 'days') > 0);
     };
+
+    this.getUpgradeProductName = (planName, ovhSubsidiary) => OvhHttp
+      .get('/order/catalog/public/baremetalServers', {
```

Comment:
To uniformize and use only one library, you should add a call to `ovh-api-services` if it doesn't already exists

Reply:
Since it's for a temporary test for now, I think we could just let it like that for now ;)

Reply:
Temporary doesn't mean that the endpoint couldn't be added to `ovh-api-services`.
Could you please taking this in account?
Thanks!